pkg/core/app/app.go

```diff
@@ -69,12 +69,13 @@ func (a *App) Setup(_ context.Context) error {
 	}
 	a.docker = d
 
-	if (a.kind == utils.Docker || a.kind == utils.DockerCompose) && IsDetachMode(a.cmd) {
-		return fmt.Errorf("detach mode is not allowed in Keploy command")
+	if (a.kind == utils.DockerStart || a.kind == utils.DockerRun || a.kind == utils.DockerCompose) && isDetachMode(a.logger, a.cmd, a.kind) {
```

Comment:
rather than this use a separate function for eg: 
```
func isDockerKind(kind utils.CmdType) bool {
    return kind == utils.DockerStart || kind == utils.DockerRun || kind == utils.DockerCompose
}
```

Reply:
Done